graspologic/partition/leiden.py

```diff
@@ -277,6 +278,13 @@ def leiden(
         if it is found to be a directed graph. If you know it is undirected and wish to
         avoid this scan, you can set this value to ``False`` and only the lower triangle
         of the adjacency matrix will be used to generate the weighted edge list.
+    trials : int
```

Comment:
I also don't want to let this hold up the PR, but this seems like something we should add to the stack of #755 and consider how we name parameters like this in the future. This exact thing happens in `match` and `cluster` at a minimum and there we call it `n_init`.